apps/assets/serializers/domain.py

```diff
@@ -27,7 +27,7 @@ class Meta:
         model = Domain
         fields_mini = ['id', 'name']
         fields_small = fields_mini + ['comment']
-        fields_m2m = ['assets', 'gateways', 'assets_amount']
+        fields_m2m = ['assets', 'gateways', 'labels', 'assets_amount']
         read_only_fields = ['date_created']
         fields = fields_small + fields_m2m + read_only_fields
         extra_kwargs = {
```

Comment:
I cannot analyze this code without understanding the specifics of its contents. It looks like it defines a Django model, however, I need to see additional context (specific fields, model definition, etc.) to provide meaningful feedback on the structure and functionality.